internal/runners/activate/activate.go

```diff
@@ -49,24 +46,6 @@ func (r *Activate) Run(params *ActivateParams) error {
 	return r.run(params, activationLoop)
 }
 
-func sendProjectIDToAnalytics(namespace *project.Namespaced, configFile string) {
-	names, fail := project.ParseNamespaceOrConfigfile(namespace.String(), configFile)
-	if fail != nil {
-		logging.Debug("error resolving namespace: %v", fail.ToError())
-		return
-	}
-
-	platProject, fail := model.FetchProjectByName(names.Owner, names.Project)
-	if fail != nil {
-		logging.Debug("error getting platform project: %v", fail.ToError())
-		return
-	}
-	projectID := platProject.ProjectID.String()
-	analytics.EventWithLabel(
-		analytics.CatBuild, analytics.ActBuildProject, projectID,
-	)
-}
```

Comment:
Leave the function and just remove the event, Martin is going to need this function for his PR.

Reply:
Not addressing this as per slack discussion.